metric_metadata/cassandra/cassandra.go

```diff
@@ -130,42 +129,33 @@ func (db *cassandraDatabase) AddMetricNames(metrics []api.TaggedMetric) error {
 	queryInsert := "INSERT INTO metric_names (metric_key, tag_set) VALUES (?, ?)"
 	queryUpdate := "UPDATE metric_name_set SET metric_names = metric_names + ? WHERE shard = ?"
 
-	c := make(chan *gocql.Query, 10)
-	done := make(chan bool)
-	go func() {
```

Comment:
It's not clear to me why a separate goroutine was spawned here- it's simpler to execute the queries synchronously.

Unless I'm mistaken, preparing the queries (calling `Bind`, but not `Exec`) should take almost no time, so trying to do this concurrently with execution shouldn't have any improvement for performance.